content/get-started/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Get Started with Webpack
+contributors:
+  - bebraw
+sort: 3
+---
+
+## Getting Started
+
+webpack is a tool to build javascript modules in your application. To start using `webpack` from its [cli](/api/cli) or [api](/api/node), follow the [Installation instructions](/get-started/install-webpack).
+webpack simplifies your workflow by quickly constructing a dependency graph of your application and bundling them in the right order. webpack can be configured to customise optimisations to your code, to split vendor/css/js code for production, have a hot reloaded dev server and many such cool features. Learn more about [why you should use webpack](/get-started/why-webpack).
+
+## Creating a bundle
+
+Create a demo directory to try out webpack. [Install webpack](/get-started/install-webpack).
+
+```bash
+mkdir webpack-demo && cd webpack-demo
+npm init -y
+npm install --save-dev webpack
+webpack --help //Shows a list of valid cli commands
+npm install --save lodash
+```
+
+Now create an `index.js` file.
+
+```javascript
+import _ from 'lodash';
+
+function component () {
+  var element = document.createElement('div');
+
+  element.innerHTML = _.map(['Hello','webpack'], function(item){
+    return item + ' ';
+  });
+
+  return element;
+};
+
+document.body.appendChild(component());
+```
+
+To run this piece of code, one usually has the below html
+
+```html
+<html>
+  <head>
+      <title>Webpack demo</title>
+      <script src='https://unpkg.com/lodash@4.16.6' type='text/javascript'/>
+      <script src='index.js' type='text/javascript'/> 
+  </head>
+  <body>
+  </body>
+</html>
+```
```

Comment:
This part was a bit confusing for me. You used `import` in the JS frament but added lodash via script tag in the HTML fragment.

Maybe it's a bit clearer if you first show JS without import + HTML with script tag. And than JS with import (maybe as diff) and HTML without script tag. This way you have two complete examples, first without webpack, second with webpack.